src/agent/ncp_openthread.cpp

```diff
@@ -60,6 +60,11 @@ using std::chrono::steady_clock;
 namespace otbr {
 namespace Ncp {
 
+otCliCommand ControllerOpenThread::sRegionCommand = {
```

Comment:
constexpr?

Reply:
`const` will be more proper for these callback structs.